### PR TITLE
Add Skyhook projects to GSoC as discussed under IRIS-HEP

### PR DESCRIPTION
**Pull request by jlefevre**
Skyhook gsoc project ideas for your consideration.  I added our proposal under IRIS-HEP project, but should I also add our org (CROSS/UCSC) as well?  We do not yet have an entry in the orgs dir, so was not sure.  Please let me know. Thank you,
Jeff

### CODE REVIEW

**Comment by aflp91** [comment]
> but should I also add our org (CROSS/UCSC) as well? We do not yet have an entry in the orgs dir, so was not sure. Please let me know. Thank you,
> Jeff

Yes, I think so; use the [orgs pattern](https://raw.githubusercontent.com/hep-sf/hep-sf.github.io/master/_gsocorgs/2020/cern.md).

File: _gsocproposals/2020/proposal_IRIS-HEPskyhookdm.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * Implement compaction of multiple formatted sub-partitions within an object into a single partition.  This could also incorporate compression. [Github Issue](https://github.com/uccross/skyhookdm-ceph/issues/33)
+ * Extend current aggregation method to include sort/groupby for formatted database partitions in objects [Github Issue](https://github.com/uccross/skyhookdm-ceph/issues/23)
+ 
+## Expected results
+The task should be fully functional for data stored in both row and column formats (Flatbuffers and Arrow).  Larger scale experiments of 1--32 node clusters can be conducted on [NSF Cloudlab](https://www.cloudlab.us) to evaluate performance.
+
+## Requirements
+C++, some Python, storage systems or database background preferred
+
+## Mentors
```
Comment (xvallspl) [inline _gsocproposals/2020/proposal_IRIS-HEPskyhookdm.md]: You will have to update mentors.md as well

File: _gsocproposals/2020/proposal_IRIS-HEPskyhookdm.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  * [JeffLeFevre](mailto:jlefevre@ucsc.edu)
```
Comment (xvallspl) [inline _gsocproposals/2020/proposal_IRIS-HEPskyhookdm.md]: Space missing?

File: _gsocproposals/2020/proposal_IRIS-HEPskyhookdm.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+title: Extend SkyhookDM programmable object storage with statistics, sort/aggregate, or data compaction functions.
+layout: gsoc_proposal
+project: IRIS-HEP
+year: 2020
+organization: CROSS/UC Santa Cruz
```
Comment (xvallspl) [inline _gsocproposals/2020/proposal_IRIS-HEPskyhookdm.md]: You will have to add the organization as well

Comment (agheata) [reply _gsocproposals/2020/proposal_IRIS-HEPskyhookdm.md]: The *organization:* field name needs to be compact and without special characters, e.g CROSS-UCSC and has to match the one in _gsocorgs/2020/cross-ucsc.md (note the file name for the org file). the logo file name used in cross-ucsc.md should be then called CROSS-UCSC-logo.png (the logo file needs to be put in the images folder

File: _gsocproposals/2020/proposal_IRIS-HEPskyhookdm.md (hunk 2 above)
Comment (jlefevre) [reply _gsocproposals/2020/proposal_IRIS-HEPskyhookdm.md]: Fixed.

File: _gsocproposals/2020/proposal_IRIS-HEPskyhookdm.md (hunk 1 above)
Comment (jlefevre) [reply _gsocproposals/2020/proposal_IRIS-HEPskyhookdm.md]: done.

File: _gsocproposals/2020/proposal_IRIS-HEPskyhookdm.md (hunk 3 above)
Comment (jlefevre) [reply _gsocproposals/2020/proposal_IRIS-HEPskyhookdm.md]: done.

File: _gsocorgs/2020/cross.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Center for Research in Open Source Software"
+author: "Jeff LeFevre"
+layout: default
+organization: CROSS
+logo: CROSS-logo.png
+description: |
+  The Center for Research in Open Source Software (www.cross.ucsc.edu), part of the Baskin School of Engineering at UC Santa Cruz, was founded based on the success of former UCSC student Sage Weil and the Ceph storage system.  At CROSS, we create open source leaders by blending open source software strategies with graduate-level research. CROSS turns OSS prototypes into successful open source software projects.
```
Comment (aflp91) [inline _gsocorgs/2020/cross.md]: It seems me a link would be more readable…!  [Center for Research in Open Source Software](www.cross.ucsc.edu)

**Comment by agheata** [comment]
all good, merging

**Pull request merged**